client/landing/stepper/declarative-flow/start-writing.ts

```diff
@@ -83,7 +90,7 @@ const startWriting: Flow = {
 				state: AssertConditionState.CHECKING,
 				message: `${ flowName } requires a logged in user`,
 			};
-		} else if ( currentUserSiteCount && currentUserSiteCount > 0 ) {
+		} else if ( userAlreadyHasSites && ! isLaunchpad ) {
```

Comment:
We prevent users with existing sites from entering the `start-writing` flow so they do not accidentally create a bunch of new sites. However, we need an exception for the Launchpad, because by the time we show them the Launchpad, they already have 1 site (the one we just created).

Reply:
This would be good to add as a comment if anyone else comes across this redirect code.

```suggestion
		} else if ( userAlreadyHasSites && ! isLaunchpad ) {
		    // Redirect users with existing sites out of the flow as we create a new site as the first step in this flow.
			// This prevents a bunch of sites being created accidentally
```